ironfish-cli/src/commands/wallet/balances.ts

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { BufferUtils, CurrencyUtils, GetBalancesResponse } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+// TODO(rohanjadvani): Remove this after assets are added to the wallet
+type Balance = GetBalancesResponse['balances'][number] & {
+  name: string
+}
+
+export class BalancesCommand extends IronfishCommand {
+  static description = `Display the account's balances for all assets`
+
+  static flags = {
+    ...RemoteFlags,
+    all: Flags.boolean({
+      default: false,
+      description: `Also show unconfirmed balance, head hash, and head sequence`,
+    }),
+    confirmations: Flags.integer({
+      required: false,
+      description: 'Minimum number of blocks confirmations for a note',
+    }),
+  }
+
+  static args = [
+    {
+      name: 'account',
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
+      required: false,
+      description: 'Name of the account to get balances for',
+    },
+  ]
+
+  async start(): Promise<void> {
+    const { flags, args } = await this.parse(BalancesCommand)
+    const client = await this.sdk.connectRpc()
+
+    const account = args.account as string | undefined
+    const response = await client.getAccountBalances({
+      account,
+      minimumBlockConfirmations: flags.confirmations,
+    })
+    this.log(`Account: ${response.content.account}`)
+
+    let showHeader = true
```

Comment:
You don't need this because you're not streaming the responses back. You can just render the whole table.

Reply:
Fixed